jujugui/static/gui/src/app/components/unit-details/test-unit-details.js

```diff
@@ -59,19 +59,48 @@ describe('UnitDetails', function() {
         unit={fakeUnit} />);
 
     assert.deepEqual(output.props.children[0],
```

Comment:
I copied this from other tests in this file. The code base is littered with these? Are they not right?